config/initializers/global_functions.rb

```diff
@@ -20,4 +20,8 @@ def development_or_test?
 
 def production_or_staging?
   staging? || production?
+end
+
+def ci?
+  # [ENV['CODESHIP'], ENV['CI']].any? &:present?
```

Comment:
CI meants Continuous Integration. Not travis-specific

1. It configurates the `rspec-retry` gem
2. Codeship sets an env var named `CODESHIP`. What `ci?` does is to determine whether code is being run by a CI provider. Either `ENV['CODESHIP']` or `ENV['CI']` would prove that